.travis.yml

```diff
@@ -1,12 +1,25 @@
-sudo: false
-
 language: d
+sudo: false
+dist: trusty
 
 addons:
-    apt:
-        packages:
-            gcc-multilib
+  apt:
+    packages:
+      - pkg-config
+      - gcc-multilib
+
+install:
+  - pyenv global system 3.6
+  - pip3 install 'meson>=0.45'
+  - mkdir .ntmp
+  - curl -L https://github.com/ninja-build/ninja/releases/download/v1.8.2/ninja-linux.zip -o .ntmp/ninja-linux.zip
+  - unzip .ntmp/ninja-linux.zip -d .ntmp
+
+before_script:
+  export PATH=$PATH:$PWD/.ntmp
 
 script:
-- git submodule update --init --recursive
-- make -B -C test/
+  - meson build && ninja -j8 -C build
+  - ninja -j8 -C build test -v
+  - git submodule update --init --recursive
```

Comment:
Shouldn't the submodules cloned before running the meson tests?

Reply:
Meson will fetch missing stuff automatically from Git repositories, in case the system does not already provide the shared libraries (it will first use pkg-config to find the library, and if that fails clone the Git repo).
By initializing the submodule after the Meson test, I just ensure that the auto-fetching works and the Meson test doesn't accidentally use the submodule (it actually can't this way, but you never know...)